src/pages/guides/index.md

```diff
@@ -243,25 +243,31 @@ To prevent this from happening, Adobe I/O Events has a robust event validation p
 <InlineAlert variant="info" slots="text"/>
 Adobe strongly encourages validating your webhook deliveries using this new mechanism to avoid processing "events" received from malicious third-party actors and make sure your webhook continues to receive events.
 
-**Improved and Resilient Security Verification for Webhook Events**
+### Improved and Resilient Security Verification for Webhook Events
 
 <InlineAlert variant="info" slots="text"/>
 This feature is scheduled to be rolled out on 10th February, 2022
 
 For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
 
-- sending additional field of recipient_client_id as part of your event payload
-- signing the event payloads digitally and sending those digital signatures as request headers to your webhook
+- Adobe I/O Events sends an additional field of `recipient_client_id` as part of your event payload.
+- The event payload is signed digitally using a fixed public/private key pair generated by Adobe I/O Events. The digital signature is sent as a webhook request header.
+- Adobe I/O Events sends the relative path of public key, which is served from our fixed Adobe domain [adobeioevents.com](https://adobeioevents.com), as webhook request headers.
 
-I/O Events generates two public-private key pairs and sign your event payload using digital signatures following the steps below
+**How it strengthens security**
+
+- Adobe I/O Events uses the same public/private key pair generated by itself to sign all event payloads flowing through it's pipeline. The public keys are also hosted at it's own domain i.e. [adobeioevents.com](https://adobeioevents.com). This ensures that any malicious user can't use his own arbitrary public/private key pair to sign an event payload and send forged requests with his random public key url to your webhook endpoint. As, now the webhook consumer is using only the Adobe I/O Events generated public key served via the fixed Adobe domain and provided to him via the request header to verify the signature.
```

Comment:
We have already mentioned in the first line that we are using a fixed domain, so all the other information seems redundant. Can we please remove it?
```suggestion
- Adobe I/O Events uses the same public/private key pair generated by itself to sign all event payloads flowing through it's pipeline. The public keys are also hosted at it's own domain i.e. [adobeioevents.com](https://adobeioevents.com). This ensures that any malicious user can't use his own arbitrary public/private key pair to sign an event payload and send forged requests to your webhook endpoint.
```